site/releases/2024-feb-14/highlights.qmd

```diff
@@ -0,0 +1,236 @@
+
+<!---John6797/sc 2943/developer framework should register documentation by @johnwalz97 in [#324](https://github.com/validmind/developer-framework/pull/324)--->
+#### Developer framework documentation inputs
+
+We have added a new feature that tracks which datasets and models are used when running tests. Now, when you initialize datasets or models with `vm.init_dataset()` and `vm.init_model()`, we link those inputs with the test results they generate. This makes it clear which inputs were used for each result, improving transparency and making it easier to understand test outcomes. This update does not require any changes to your code and works with existing `init` methods.
+
+### ValidMind Platform UI (v1.13.13)
+
+<!---Update events to show users who ran the developer framework actions by @cachafla in [#532](https://github.com/validmind/frontend/pull/532)--->
+#### Update events to show users who ran the developer framework actions
+
+Update rendering of events related to the developer framework. We are now showing the user who ran the action instead of a generic "Developer Framework" actor name
+
+::: {.grid}
+::: {.g-col-6}
+**Before**
+
+<img src="78586ce2-4e9d-4613-9738-b09db49c2d5e.png" alt="Old indicator that says that the developer framework logged an event" style="width: 100%; border: 1px solid #333; box-shadow: 5px 5px 5px #ccc, -5px 5px 5px #ccc; border-radius: 5px;">
+
+:::
+
+::: {.g-col-6}
+**After**
+
+<img src="8b643ea9-61c6-445e-bf2a-422d94193ffc.png" alt="New indicator that specifies the user who made updates through the developer framework" style="width: 100%; border: 1px solid #333; box-shadow: 5px 5px 5px #ccc, -5px 5px 5px #ccc; border-radius: 5px;">
+:::
+:::
+
+
+<!---Nrichers/sc 2163/revisit once again the getting started page by @nrichers in [#543](https://github.com/validmind/frontend/pull/543)--->
+#### Simplified instructions for developers
+
+We simplified the instructions for getting started with the ValidMind Developer Framework in the Platform UI. These instructions tell you how to use the code snippet for your documentation project with your own model or with one of our code samples.
+
+<img src="bf9b42d3-c74f-41f9-97b5-52911e33eb0d.png" alt="New indicator that specifies the user who made updates through the developer framework" style="width: 80%; border: 1px solid #333; box-shadow: 5px 5px 5px #ccc, -5px 5px 5px #ccc; border-radius: 5px;">
+
+## Enhancements
+
+<!---Panchicore/sc 2799/model owner should be able to edit model by @panchicore in [#562](https://github.com/validmind/frontend/pull/562)--->
+Model owner can edit model details page
+: Model owners can now edit the values for fields displayed on the model details page. Previously it was only possible to edit custom fields defined by your organization.
+
+<!---Reducing load time by using localstorage for Auth0 by @even-steven in [#570](https://github.com/validmind/frontend/pull/570)--->
+Reduced load times for the ValidMind Platform UI
```

Comment:
```suggestion
Enhanced page load times for the ValidMind Platform UI
```

Reply:
Minor comment: my brain interprets "enhanced load times" in a weird way since load time is a measure of quantity. An "enhanced load time" gives me a confusing measure of quantity 🤔.

Reply:
Yeah, I'm not 100% that we've _enhanced_ load times. I might just change this to `Improved performance for the ...`.